Sources/StreamChat/Query/Sorting/ChannelMemberListSortingKey_Tests.swift

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+@testable import StreamChat
+@testable import StreamChatTestTools
+import XCTest
+
+final class ChannelMemberListSortingKey_Tests: XCTestCase {
+    func test_sortDescriptor_keyPaths_areValid() throws {
+        // Put all `ChannelMemberListSortingKey`s in an array
+        // We don't use `CaseIterable` since we only need this for tests
+        let sortingKeys = [ChannelMemberListSortingKey.createdAt]
+        
+        // Iterate over keys...
+        for key in sortingKeys {
+            switch key {
+            case .createdAt:
+                // ... and make sure all keys correspond to a valid KeyPath
+                XCTAssertEqual(key.rawValue, NSExpression(forKeyPath: \MemberDTO.memberCreatedAt).keyPath)
```

Comment:
```swift
NSExpression(forKeyPath: \MemberDTO.memberCreatedAt).keyPath
```
is a hacky way to get String from a keyPath
https://stackoverflow.com/a/51376768